examples/simulation-pytorch/sim.py

```diff
@@ -0,0 +1,223 @@
+import argparse
+from collections import OrderedDict
+from typing import Dict, Tuple, List
+
+import torch
+import torchvision
+from torch.utils.data import DataLoader, random_split
+
+import flwr as fl
+from flwr.common import Metrics
+from flwr.common.typing import Scalar
+
+from utils import Net, train, test, get_mnist
+
+
+parser = argparse.ArgumentParser(description="Flower Simulation with PyTorch")
+
+parser.add_argument(
+    "--num_cpus",
+    type=int,
+    default=1,
+    help="Number of CPUs to assign to a virtual client",
+)
+parser.add_argument(
+    "--num_gpus",
+    type=float,
+    default=0.0,
+    help="Ratio of GPU memory to assign to a virtual client",
+)
+parser.add_argument("--num_rounds", type=int, default=10, help="Number of FL rounds.")
+
+NUM_CLIENTS = 100
+
+
+# Flower client, adapted from Pytorch quickstart example
+class FlowerClient(fl.client.NumPyClient):
+    def __init__(self, trainset, valset):
+        self.trainset = trainset
+        self.valset = valset
+
+        # Instantiate model
+        self.model = Net()
```

Comment:
It'd be slightly better / more consistent to do the model instantiation in the `client_fn`, and then pass the model instance to the constructor